pkgs/development/libraries/isl/0.17.1.nix

```diff
@@ -1,24 +1,11 @@
 { lib, stdenv, fetchurl, gmp }:
```

Comment:
```suggestion
{ callPackage, lib, stdenv, fetchurl, gmp }:
```

Some of the older versions are missing this callPackage arg

Reply:
whoops, that slipped. Thanks!